### DISCUSSION TITLE
IsAuthorized for TabPanel

### DISCUSSION THREAD

**Discussion by wcrisafu** [oqtane/oqtane.framework#797] [Ideas] [answered]
I am playing with the various controls and I was wondering if there is a plan to implement the "IsAuthorized" in the TabPanel, just like we have in the ActionDialog and ActionLink. I think enabling the tabs visibility based on the user security level, could be very useful.

**Comment by sbwalker** [comment]
There is no IsAuthorized public parameter on ActionLink or ActionDialog... I assume what you are actually referring to is the SecurityAccessLevel parameter? This property allows you to specify the types of users who can access those controls. So are you asking if there can be a SecurityAccessLevel parameter on the TabPanel?

**Reply by wcrisafu** [reply]
Yes, you are correct.  The "IsAuthorized" is the function inside those two components that does the validation and sets the "_authorized" variable. I thought we could use the same logic to limit the tabs visibility (Anonymous, Edit, Admin, etc.).

**Reply by sbwalker** [reply]
I think this would be a useful enhancement

**Comment by sbwalker** [comment]
@wcrisafu This is implemented in #863

**Comment by wcrisafu** [comment, marked as answer]
Thank you!
